deflect/server/Server.cpp

```diff
@@ -125,7 +125,7 @@ class Server::Impl : public QTcpServer
             connect(frameDispatcher, &FrameDispatcher::sourceRejected, worker,
                     &ServerWorker::closeConnection);
             connect(worker, &ServerWorker::receivedTile, frameDispatcher,
-                    &FrameDispatcher::processTile);
+                    &FrameDispatcher::processTile, Qt::DirectConnection);
```

Comment:
can't judge about concurrent access that happens now that the tiles are added one a different thread. they're probably save, as the finish of a frame is still queued?